### ISSUE TITLE
Lighthouse Audit: Link text/alt text missing (footer links)

### ISSUE THREAD

**Issue by bkinnamon** [bkinnamon/pomodoro#15]

**Comment by bkinnamon** [comment]
https://dequeuniversity.com/rules/axe/3.1/link-name?application=lighthouse
